applications/mobu/values.yaml

```diff
@@ -26,6 +26,9 @@ config:
   # Each flock listed will be automatically started when mobu is started.
   autostart: []
 
+  # -- GitHub app integration options.
+  github: {}
```

Comment:
Add "# @default -- See `values.yaml`" here to prevent injecting the object into the docs.